src/expandableCalendar/week.tsx

```diff
@@ -15,7 +15,7 @@ import Day from '../calendar/day/index';
 export type WeekProps = CalendarProps;
 
 const Week = (props: WeekProps) => {
-  const {theme, current, firstDay, hideExtraDays, markedDates, onDayPress, style: propsStyle, numberOfDays = 1, timelineLeftInset} = props;
+  const {theme, current, firstDay, hideExtraDays, markedDates, onDayPress, onLongPress, style: propsStyle, numberOfDays = 1, timelineLeftInset} = props;
```

Comment:
The name of the props is wrong - you need Calendar's `onDayLongPress`

Reply:
Thanks for the review @Inbal-Tish - i lack knowledge about the sematic difference of onLongPress vs onLongDayPress, what i did though was modifying it in my local repository where i had react-native-calendars installed as a dependency (directly modified it in the node_modules), and the thing i wanted to achieve only worked, when i modified it as in the PR. When i use onDayLongPress instead, it - for some reason - does not get triggered if i long press on the week component's day elements.
Maybe i am missing something, can u help me investigate this?